### PR TITLE
[transaction builder generator + libra-doc-tool] update serde-generate to 0.10.0

### PR DESCRIPTION
**Pull request by matbd**
## Motivation

* Follow up with recent API stabilization changes in `serde-generate`.
* Fix comments
* Simplify stdlib tool by addressing TODO and removing `post_process_add_proptest_derivations` (see https://github.com/facebookincubator/serde-reflection/issues/32)

## Test Plan

CI + 
```
cargo x test -p transaction-builder-generator -- --ignored
```

## Related PRs

https://github.com/facebookincubator/serde-reflection/pull/39

### CODE REVIEW

File: common/libradoc/src/lib.rs [hunk 1]
```diff
@@ -3,8 +3,20 @@
...
+use serde_generate::{rust, CodeGeneratorConfig};
+use serde_reflection::Registry;
 use std::{collections::BTreeMap, io::BufRead};
 
+pub fn quote_container_definitions(
+    registry: &Registry,
+) -> std::result::Result<BTreeMap<String, String>, Box<dyn std::error::Error>> {
+    let config = CodeGeneratorConfig::new("crate".to_string()).with_serialization(false);
+    let generator = rust::CodeGenerator::new(&config)
+        .with_derive_macros(Vec::new())
```
Comment (bmwill) [inline common/libradoc/src/lib.rs:15]: Seems odd to need to pass in an empty Vec here, presumably to remove the default derives?

Comment (matbd) [reply common/libradoc/src/lib.rs:15]: That's right!

Comment (matbd) [reply common/libradoc/src/lib.rs:15]: Adding a comment

**Pull request closed without merging**